core/src/sql/statements/remove/table.rs

```diff
@@ -58,6 +63,37 @@ impl RemoveTableStatement {
 			v => v,
 		}
 	}
+
+	async fn terminate_lives(&self, ctx: &Context<'_>, opt: &Options) -> Result<(), Error> {
+		// Check if we can send notifications
+		if let Some(chn) = &opt.sender {
+			// Get all live queries for this table
+			let lives = ctx.tx().all_tb_lives(opt.ns()?, opt.db()?, &self.name).await?;
```

Comment:
This won't be safe, because there can be a transaction that creates a live query as this transaction is deleting it.
We need some way to conflict new live query transactions.
We can merge as is and create a ticket for this.
Please link a ticket to this comment once created.